sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -381,7 +381,7 @@ def test_lambertw():
     assert LambertW(2, evaluate=False).is_real
     p = Symbol('p', positive=True)
     assert LambertW(p, evaluate=False).is_real
-    assert LambertW(p - 1, evaluate=False).is_real is None
+    assert LambertW(p - 1, evaluate=False).is_real is False
```

Comment:
when we call ```LambertW(p - 1, evaluate=False).is_real```--->
https://github.com/sympy/sympy/blob/master/sympy/functions/elementary/exponential.py#L842
--->https://github.com/sympy/sympy/blob/master/sympy/core/add.py#L612 this function give wrong answer ```c = -1 and a = p + exp(-1)``` but right answer is ```c = -1 + exp(-1) and a = p``` so we need to modify ```as_coeff_Add()``` function.